hack/tilt/values.dev.yaml

```diff
@@ -26,11 +26,22 @@ api:
         type: mockCallback
       probes:
         enabled: false
+    # Just testing different value formats, will revert before merge
     admins:
+      # JSON format with arrays
       claims:
-        email:
-        # This email claim is hard-coded in the Dex mockCallback connector
-        - kilgore@kilgore.trout
+        {
+          "email": ["kilgore@kilgore.trout", "user@inbox.com"],
+          "cognito:groups": ["devops"]
+        }
```

Comment:
I know these are changes you are just using for testing and intend to revert, so this is only an FYI as a point of reference. YAML is a superset of JSON that enables additional (simpler) syntax, so all valid JSON is _also_ valid YAML. I mention this because I think you may have used this syntax thinking it tests a different path through the code than that taken by the data starting on line +40, but it doesn't. i.e. Feel free to stick exclusively to the nicer syntax. It won't make this any less thorough.